dockstore-webservice/src/main/java/io/dockstore/webservice/resources/TokenResource.java

```diff
@@ -234,7 +234,10 @@ public Token addQuayToken(@ApiParam(hidden = true) @Parameter(hidden = true, nam
     private void checkIfAccountHasBeenLinked(String username, TokenType tokenType) {
         Token existingToken = tokenDAO.findTokenByUserNameAndTokenSource(username, tokenType);
         if (existingToken != null) {
-            String msg = "A '" + tokenType.toString() + "' token already exists on Dockstore for the account '" + username + "'";
+            User dockstoreUser = userDAO.findById(existingToken.getUserId());
+            String msg = "The '" + tokenType.toString() + "' account '" + username
+                + "' is already linked to another Dockstore account with the name '"
```

Comment:
Tied to my previous comment, but I think the more interesting thing is what log-in the other account used, not the Dockstore username (even though they might be the same).